ui/forms.py

```diff
@@ -36,14 +36,10 @@ def __init__(self, *args, **kwargs):
     def clean_course_file(self):
         """Only certain extensions are allowed."""
         upload = self.cleaned_data["course_file"]
-        log.debug("checking filename %s", upload.name)
         for ext in (".zip", ".tar.gz", ".tgz"):
```

Comment:
I think the form validation is the best place for it, it's not used anywhere else, and that would require another import.

However, it should probably be in the UI on the import page user-facing documentation. That would justify having it in a setting somewhere else. Based on @carsongee's comments for other parts of this PR that would be out of scope, so please make a separate issue for that.